src/components/data-table/data-table-story.mdx

```diff
@@ -162,7 +162,7 @@ html`
 
 Adding `sort-direction` attribute to `<bx-table-header-cell>` shows the sorting UI in the table header cell.
 
-When user attempts to change sorting, `bx-table-header-cell-sort` event is fired, with the following properties in the event details:
+The logic of sorting needs to be handled on by the application code. When user attempts to change sorting, `bx-table-header-cell-sort` event is fired, with the following properties in the event details:
```

Comment:
same as above?

Reply:
Added here since we do have an example for sorting.